source/_posts/2017/2017-02-15-minimalistic-way-to-create-your-first-nette-extension.md

```diff
@@ -121,7 +120,7 @@ final class FileSystemExtension extends CompilerExtension
         // this method loads servcies from config and registers them do Nette\DI Container
         Compiler::loadDefinitions(
             $this->getContainerBuilder(),
-            $this->loadFromFile(__DIR__.'/../config/services.neon')['services']
+            $this->loadFromFile(__DIR__.'/../config/services.neon')
```

Comment:
Toto teď nebude funkční :) Mrkni do docku https://api.nette.org/2.4/source-DI.Compiler.php.html#257 Metoda `loadDefinitions` očekává, že ji předáš právě pole služeb, ale `loadFromFile` ti vrátí strukturu `[['services' => [..., ...]]` což je problém. Taky jsem se na to nachytal... :) Proto jsem ti posílal tu ukázku i s kódem kde byl `isset` atd.